clinics/walgreens.py

```diff
@@ -26,7 +26,7 @@ def get_locations(self):
                     "%Y-%m-%d"
                 )
             },
-            "radius": max(20, int(os.environ["RADIUS"])),
+            "radius": min(25, int(os.environ["RADIUS"])),
```

Comment:
I swear that I added the max before it was erroring if I requested a radius higher than 20.

But I'm not able to reproduce that now. I'd be in favor of passing the radius without a min or max.

Reply:
We don't need to cast to int anymore since we're not taking a max
```suggestion
            "radius": int(os.environ["RADIUS"]),
```

Reply:
Radius larger than 25 still gives me this:
```
{"errors":[{"code":"FC_901_ValidationFailed","message":"/radius Invalid, "}]}
```

Reply:
Ohhhh I see what happened. It was 20 originally, I changed to the env var `"25"`, which was invalid.
I thought they didn't support higher than 20, when really they don't support a string value, it just needs an int cast.

Reply:
Min not needed though
```suggestion
            "radius": int(os.environ["RADIUS"]),
```

Reply:
I'm also seeing it error for radius over 25, but we need a max, not a min
```suggestion
            "radius": max(25, int(os.environ["RADIUS"])),
```